glslc/README.asciidoc

```diff
@@ -390,10 +385,18 @@ output.
 The `GL_GOOGLE_cpp_style_line_directive` extension is implicitly turned on by
 the `GL_GOOGLE_include_directive` extension.
 
-It is invalid to have any '\' or '/' characters in any filename provided to
-an `#include` directive. The presence of any such characters may lead to
-unexpected behavior on different systems.
-
-Furthermore it is not possible to escape any characters in an `#include`
-directive and as such, any special characters that would have to be escaped in
-C may not show up in file names either.
+The file argument to `#include` must be enclosed in double quotes.  It must be a
+relative path, using whatever path separator the OS supports.  However, the last
+path element -- the name of the file itself -- must not contain either '/' or
+'\', regardless of which path separator is used.  For example, the following is
+invalid: `#include "dir\subdir\subsubdir\filen/ame.vert"`.  This will not be
```

Comment:
This *is* a valid request, it's just that glslc will interpret it as a request for file "ame.vert" in relative path "dir\subdir\subsubdir\filen".  We should say *that* instead of saying it's invalid.